<UI version="4.0" >
 <class>FfmpegViewClass</class>
 <widget class="QWidget" name="FfmpegViewClass" >
  <property name="objectName" >
   <string notr="true">FfmpegViewClass</string>
  </property>
  <property name="geometry" >
   <rect>
	<x>0</x>
	<y>0</y>
	<width>600</width>
	<height>400</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>FfmpegView</string>
  </property>
	 <widget class="QWidget" name="centralWidget">
		 <layout class="QGridLayout" name="gridLayout">
			 <item row="0" column="0">
				 <layout class="QVBoxLayout" name="verticalLayout">
					 <item>
						 <layout class="QVBoxLayout" name="verticalLayout_2">
							 <item>
								 <widget class="QLabel" name="titleLable">
									 <property name="text">
										 <string/>
									 </property>
								 </widget>
							 </item>
							 <item>
								 <spacer name="verticalSpacer">
									 <property name="orientation">
										 <enum>Qt::Vertical</enum>
									 </property>
									 <property name="sizeHint" stdset="0">
										 <size>
											 <width>20</width>
											 <height>40</height>
										 </size>
									 </property>
								 </spacer>
							 </item>
						 </layout>
					 </item>
					 <item>
						 <layout class="QHBoxLayout" name="horizontalLayout_3">
							 <item>
								 <widget class="QSlider" name="videoProgressSlider">
									 <property name="orientation">
										 <enum>Qt::Horizontal</enum>
									 </property>
								 </widget>
							 </item>
							 <item>
								 <widget class="QLabel" name="labelTime">
									 <property name="text">
										 <string/>
									 </property>
								 </widget>
							 </item>
						 </layout>
					 </item>
					 <item>
						 <layout class="QHBoxLayout" name="horizontalLayout_2">
							 <item>
								 <spacer name="horizontalSpacer">
									 <property name="orientation">
										 <enum>Qt::Horizontal</enum>
									 </property>
									 <property name="sizeHint" stdset="0">
										 <size>
											 <width>40</width>
											 <height>20</height>
										 </size>
									 </property>
								 </spacer>
							 </item>
							 <item>
								 <widget class="QPushButton" name="btnPreview">
									 <property name="text">
										 <string/>
									 </property>
								 </widget>
							 </item>
							 <item>
								 <widget class="QPushButton" name="btnPause">
									 <property name="text">
										 <string/>
									 </property>
								 </widget>
							 </item>
							 <item>
								 <widget class="QPushButton" name="btnStop">
									 <property name="text">
										 <string/>
									 </property>
								 </widget>
							 </item>
							 <item>
								 <widget class="QPushButton" name="btnNext">
									 <property name="text">
										 <string/>
									 </property>
								 </widget>
							 </item>
							 <item>
								 <spacer name="horizontalSpacer_2">
									 <property name="orientation">
										 <enum>Qt::Horizontal</enum>
									 </property>
									 <property name="sizeHint" stdset="0">
										 <size>
											 <width>40</width>
											 <height>20</height>
										 </size>
									 </property>
								 </spacer>
							 </item>
						 </layout>
					 </item>
					 <item>
						 <layout class="QHBoxLayout" name="horizontalLayout_4">
							 <item>
								 <widget class="QLineEdit" name="lineEdit"/>
							 </item>
							 <item>
								 <widget class="QPushButton" name="btnOpenUrl">
									 <property name="text">
										 <string>打开网络文件</string>
									 </property>
								 </widget>
							 </item>
							 <item>
								 <widget class="QPushButton" name="btnOpenLocal">
									 <property name="text">
										 <string>打开本地文件</string>
									 </property>
								 </widget>
							 </item>
						 </layout>
					 </item>
				 </layout>
			 </item>
		 </layout>
	 </widget>
 </widget>
 <layoutDefault spacing="6" margin="11" />
 <pixmapfunction></pixmapfunction>
 <connections/>
</UI>
